src/main/java/me/desht/pneumaticcraft/common/thirdparty/cofhcore/ThermalFoundationExplosiveLaunching.java

```diff
@@ -41,6 +41,13 @@ public static void registerExplosiveLaunchBehaviour() {
         register("phyto_grenade");
         register("nuke_grenade");
 
+        // Adds thermal charges to launch map
+        // TODO: Find a way to get these to despawn after being launched, currently they float around forever
```

Comment:
Are these coded like vanilla fireballs?  I did fix a similar problem with those a while back, in commit 7f02173b67fe06f03e3a57d79e87d73059fc9bd2